web/src/PredictImpactMode.svelte

```diff
@@ -54,12 +55,44 @@
     </p>
     <p>
       Thicker roads have more traffic after edits, relative to the max count for
-      any road: {data.max_count.toLocaleString()}
+      any road: {impactGj.max_count.toLocaleString()}
     </p>
+
+    <SequentialLegendBucketed
+      colorScale={divergingScale}
+      buckets={["0%", "50%", "same", "150%", "200%"]}
+    />
   </div>
 
   <div slot="map">
-    <GeoJSON {data} generateId>
+    <GeoJSON data={neighbourhoods}>
+      <FillLayer
+        {...layerId("neighbourhood-boundaries", false)}
+        filter={["==", ["get", "kind"], "boundary"]}
+        paint={{
+          "fill-color": "grey",
+          "fill-opacity": 0.5,
+        }}
+      />
+    </GeoJSON>
+
+    <GeoJSON data={impactGj} generateId>
+      <LineLayer
+        {...layerId("predict-impact-outline")}
+        paint={{
+          "line-width": [
+            "interpolate",
+            ["linear"],
+            ["get", "after"],
+            0,
+            1.5 * minRoadWidth,
```

Comment:
Line outlines in maplibre have to be done with a second layer underneath that's thicker